spring-boot-admin-server-ui/src/test/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiAutoConfigurationTest.java

```diff
@@ -97,7 +99,7 @@ public void contextPathIsRespectedInIncludedRoutes(String routeIncludes) {
 						HomepageForwardingFilter bean = context.getBean(HomepageForwardingFilter.class);
 						bean.filter(serverWebExchange, webFilterChain);
 
-						verify(serverWebExchange, atMostOnce()).mutate();
+						verify(serverWebExchange, times(1)).mutate();
```

Comment:
This is just a bit more strict condition. If it's not present, my testcases will pass even without implementation

Reply:
yes this was hiding the bug from above